fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckFooter.kt

```diff
@@ -41,8 +40,10 @@ fun ReviewQualityCheckFooter(
         ),
         linkTextState = LinkTextState(
             text = poweredByLinkText,
-            url = FOOTER_LINK,
-            onClick = onLinkClick,
+            url = ReviewQualityCheckNavigationMiddleware.LinkType.PoweredBy().url,
+            onClick = {
```

Comment:
🔍 `onclick = onLinkClick,`
reads nicer 😄

Reply:
I agree, but the `onClick` from `LinkState` accepts a `String` parameter and `onLinkClick` has none.

Reply:
Oh yeah you're right!